doc/sphinx/Clusters_from_Scratch/fencing.rst

```diff
@@ -27,7 +27,7 @@ In order to guarantee the safety of your data [#]_, fencing is enabled by defaul
 .. NOTE::
 
     It is possible to tell the cluster not to use fencing, by setting the
-    **stonith-enabled** cluster option to false:
+    **stonith-enabled** cluster property to false:
```

Comment:
Interesting conflict ... pcs calls these cluster properties, but Pacemaker Explained distinguishes between "CIB properties" set as XML attributes of the cib element and "cluster options" set in their own elements. But since this uses pcs we can go with what you have here, and I don't think users will care anyway.

Reply:
Interesting indeed. The CIB also puts them in an element called `<cluster_property_set>`.